scripts/generate_integritee_pallets_upstream_patch.sh

```diff
@@ -44,15 +47,22 @@ cd worker
 [ ! -z "$1" ] && git checkout "$1"
 echo "generating patch ..."
 git diff $OLD_COMMIT HEAD > "$ROOTDIR/upstream.patch"
-git rev-parse --short HEAD > "$ROOTDIR/pallets/upstream_commit"
+git rev-parse --short HEAD > "$ROOTDIR/upstream_commit"
 
 echo "======================================================================="
 echo "upstream_commit is updated."
 echo "be sure to fetch the upstream to update the hashes of files."
-echo "upstream.patch is generated, to apply it, run:"
-echo '  # git am -3 --include=test-utils --include=teerex --include=teeracle --include=sidechain --include=primitives < upstream.patch'
+echo ""
+echo "upstream.patch is generated, to apply it, RUN FROM $ROOTDIR0:"
+echo "  git am -3 --directory=pallets < pallets/upstream.patch"
```

Comment:
will it find the `primitives/` and apply the patch even in the wrong dir?

Reply:
Yes. It will find. Actually I tried several ways/directories back and forth. Either way needs manually check and even move files around. To set '--directory=pallets' is the least effort.